internal/resolution/variable_sources/olm/olm.go

```diff
@@ -42,10 +62,8 @@ func (o *VariableSource) GetVariables(ctx context.Context, entitySource input.En
 		if err != nil {
 			return nil, err
 		}
-		inputVariableSources = append(inputVariableSources, rps)
+		variableSources = append(variableSources, rps)
 	}
 
-	// build variable source pipeline
-	variableSource := crdconstraints.NewCRDUniquenessConstraintsVariableSource(bundlesanddependencies.NewBundlesAndDepsVariableSource(inputVariableSources...))
```

Comment:
I had to remove this chaining within the variable source becase for off-cluster usage we need a bit different set of variable soruces (e.g. we need a variable source which creates a variable from CLI inputs).

instead of chaining I now use `NestedVariableSource` in `NewOLMVariableSource` (note after #273 it is `NewOperatorVariableSource`). `NewOLMVariableSource` is used in on-cluster operator and it is not in this diff (I did not change the calling code).


https://github.com/operator-framework/operator-controller/blob/9a1a15d7d7c7e069dab21ff5930052d13553e1fc/cmd/manager/main.go#L108C19-L108C19

But `NewCRDUniquenessConstraintsVariableSource` and `NewBundlesAndDepsVariableSource` are also used in off-cluster resolution [here within `olm.NestedVariableSource`](https://github.com/operator-framework/operator-controller/pull/246/files#diff-0e1413d2458238cf94143692a8b6990ff6d5da773e6ea780707586fb1d609409R125-R139).

I suspect `NestedVariableSource` and probably whole `composite.go` will go away once we switch to the solution proposed in the [RFC](https://docs.google.com/document/d/1PizF1uQcMrrAFsLgcx2O0Coq7mei4xe9hxpz-QRp-x0/edit#heading=h.x3tfh25grvnv).